pulser/simulation/simulation.py

```diff
@@ -241,7 +266,29 @@ def make_vdw_term() -> float:
                 vdw += U * self._build_operator('sigma_rr', q1, q2)
             return vdw
 
-        def build_coeffs_ops(basis: str, addr: str) -> list[list]:
+        def make_interaction_xy_term() -> float:
```

Comment:
This method can be included into a general `make_interaction_term` that gives the (ising/XY) interaction according to the `self._interaction` attribute